roles/installer/templates/install-config-virtualmedia.j2

```diff
@@ -75,7 +75,17 @@ platform:
 {% if ((release_version.split('.')[0]|int == 4) and (release_version.split('.')[1]|int == 6)) %}
     provisioningHostIP: {{ provisioningHostIP }}
 {% endif %}
-    bootstrapProvisioningIP: {{ bootstrapProvisioningIP }}
```

Comment:
@tonyskapunk let me test removing `bootstrapProvisioningIP` in a 4.12 deployment using DHCP to configure the bootstrapVM external interface. And see if it works. 

My understanding from this [comment](https://issues.redhat.com/browse/OCPBUGS-36869?focusedId=25463757&page=com.atlassian.jira.plugin.system.issuetabpanels:comment-tabpanel#comment-25463757) in OCPBUGS-36869 is that `bootstrapProvisioningIP` shouldn't be used when setting provisioning network disabled (which is the case when using virtual media).

`bootstrapProvisioningIP` is intended to define an IP from the provisioning network when enabled. I suspect all this time we have believed was the responsible to set the IP address in the bootstrapVM external network, but it was our DHCP server that was configuring it, because we have been using `provisioningNetwork: Disabled`

Reply:
> My understanding from this [comment](https://issues.redhat.com/browse/OCPBUGS-36869?focusedId=25463757&page=com.atlassian.jira.plugin.system.issuetabpanels:comment-tabpanel#comment-25463757) in OCPBUGS-36869 is that bootstrapProvisioningIP shouldn't be used when setting provisioning network disabled (which is the case when using virtual media).

It makes sense

> but it was our DHCP server that was configuring it, because we have been using provisioningNetwork: Disabled

:open_mouth: That would explain getting the IP, :sweat_smile: 

Thanks for the clarification